tests/core/thermal_expansion_phase_change.cc

```diff
@@ -0,0 +1,87 @@
+/**
+ * @brief Tests the thermal_conductivity_phase_change model.
+ * During a phase change the physical properties (including conductivity) of the
```

Comment:
```suggestion
 * @brief Tests the thermal_expansion_phase_change model.
 * During a phase change the physical properties (including thermal expansion) of the
```

Update the rest of the text also. It only talks about conductivity.